newm-server/src/main/kotlin/io/newm/server/features/collaboration/database/CollaborationEntity.kt

```diff
@@ -60,6 +60,18 @@ class CollaborationEntity(id: EntityID<UUID>) : UUIDEntity(id) {
             (CollaborationTable.songId eq songId) and (CollaborationTable.email.lowerCase() eq email.lowercase())
         }
 
+        fun exists(ownerId: UUID, email: String, status: CollaborationStatus): Boolean {
+            return CollaborationTable.join(
```

Comment:
I think this can be simplified using the `innerJoin` extension function on Tables.

Reply:
Good callout! Fixed here and in a couple of other places.